src/java/org/apache/cassandra/cql3/UntypedResultSet.java

```diff
@@ -359,6 +359,12 @@ public boolean getBoolean(String column)
             return BooleanType.instance.compose(data.get(column));
         }
 
+        public boolean getBoolean(String column, boolean ifNull)
+        {
+            ByteBuffer bytes = data.get(column);
+            return bytes == null ? ifNull : BooleanType.instance.compose(bytes);
+        }
+
```

Comment:
The method will no longer be needed if the other comment in this review is followed.